src/components/ActionList/ActionList.scss

```diff
@@ -83,6 +75,19 @@
     @include high-contrast-outline($border-width: var(--p-border-width-3));
   }
 
+  &:visited {
+    color: inherit;
+  }
+
+  &.active {
+    @include recolor-icon(var(--p-interactive));
+    background-color: var(--p-surface-selected);
+
+    &::before {
+      @include list-selected-indicator;
+    }
+  }
```

Comment:
@kyledurand This is that class I mentioned moving during our pairing sesh and wanted to confirm that reordering won't muck up other styles. If it can be reordered, I thought it would be better to separate `.active` and `:active` and group all the classes together and all pseudo-classes  together.